modules/jaggery-apps/portal/gadgets/account-recovery/index.jag

```diff
@@ -68,11 +68,11 @@ session.put('user', user);
             }
         }
 
-print( orderedChallanges );
-    }catch(e){
-print(e);
+        print(orderedChallanges);
+    } catch (e) {
+        print(e);
```

Comment:
Do we need to print e directly? Shouldn't it print some general error to show in the UI and log the actual error?